container/tabsapps.go

```diff
@@ -0,0 +1,220 @@
+package container
+
+import (
+	"fyne.io/fyne"
+	"fyne.io/fyne/canvas"
+	"fyne.io/fyne/layout"
+	"fyne.io/fyne/theme"
+	"fyne.io/fyne/widget"
+)
+
+// MaxAppTabs is the maximum number of visible tabs in an AppTab.
+// Any addition tabs are moved to the overflow menu.
+const MaxAppTabs = 7
```

Comment:
The number of tabs displayed should not be static - it makes more sense to display all that fit, and leave the rest in the overflow.

Reply:
I am following what was outlined in the two bugs this PR is addressing.

Reply:
It was indeed written rather specifically - I would focus more on the "UI tends to overfit" and less on the "7". In mobile, portrait mode 5 is pretty much the limit.

Reply:
Sorry - It is possible I misread this the first time, I thought it was cutting the apps at 7, not limiting it.
With that in mind is limiting it to 7 the right thing to do even on desktop when the window could be much larger?

Reply:
Yes the behaviour is that it will display as many tabs as can fit, but no more than 7. All other tabs will be be in the overflow. I don't mind having separate limits for mobile vs desktop as the screen sizes are different, but what would be a sensible limit, or should there not be a limit at all?

Reply:
I wonder if the limit is adding anything in hindsight...
It just seems like it might lead to some empty space when not required.

Reply:
Maybe this limit should only apply on mobile?

Reply:
(a grid layout is used on mobile so there wont be any empty space - tab buttons will be expanded to fill)